.github/workflows/main.yml

```diff
@@ -20,9 +20,33 @@ jobs:
         run: make check PYTHON=python3
       - name: Rename
         run: cp mavlink-routerd mavlink-routerd-libc-x86-64
+      - name: package-deb
+        run: |
+          mkdir -p .debpkg/usr/bin
+          cp mavlink-routerd .debpkg/usr/bin/mavlink-routerd
+          mkdir -p .debpkg/DEBIAN
+          export VERSION=$([[ "${{ github.ref }}" == refs/tags/* ]] && echo $(echo ${{ github.ref }} | sed s#"refs/tags/v"##) || echo "0.0.0")
+          export PKGSIZE=$(du mavlink-routerd)
+          echo "Building mavlink-router version: ${VERSION}"
+          cat > .debpkg/DEBIAN/control << EOL
+          Package: mavlink-router
+          Version: ${VERSION}
+          Section: custom
+          Priority: optional
+          Architecture: amd64
+          Essential: no
+          Installed-Size: ${PKGSIZE}
+          Maintainer: Dronecode Project
```

Comment:
mavlink-router is not part of dronecode, nor dronecode controls. "mavlink-router developers" should be sufficient here